draft-ietf-quic-tls.md

```diff
@@ -1421,7 +1421,12 @@ enable 0-RTT in TCP.
 All the restrictions on the use of 0-RTT apply, with the exception of the ALPN
 label, which MUST only change to a label that is explicitly designated as being
 compatible.  The client indicates which ALPN label it has chosen by placing that
-ALPN label first in the ALPN extension.
+ALPN label first in the ALPN extension. In order to be usable for 0-RTT,
+the NewSessionTicket MUST contain the "max_early_data" extension with the
+value 0xffffffff; the amount of data which the client can send in 0-RTT
+is controlled by the "initial_max_data" and "initial_max_stream_data"
+parameters supplied by the server. It is an error for the server to provide
+any other value for "max_early_data".
```

Comment:
Which error?

Reply:
PROTOCOL_VIOLATION perhaps?

Reply:
Yeah, no point in defining anything more specific here.  If we did that for every parameter, we'd have a million useless error codes.